Dockerfile

```diff
@@ -38,7 +37,8 @@ ENV LANGUAGE en_US.UTF-8
 
 # Add switch mirror to fix the issue.
```

Comment:
@yakutovicha Are you ok with deleting/commenting this (for me it's fine)?

If not, I suggest you just put an if/else switch here based on the `ARCH` (as for miniconda below).
Note that you will need to add the mirror for the correct ubuntu version (it's not longer bionic)

Reply:
I am definitely ok with it, as soon as the issue https://github.com/aiidalab/aiidalab-docker-stack/issues/9 does not pop up again. But since we are building things on Docker hub - it shouldn't.